rerun_py/rerun_sdk/rerun/experimental.py

```diff
@@ -9,28 +9,29 @@
 from rerun.log.experimental.blueprint import add_space_view, new_blueprint, set_auto_space_views, set_panels
 
 __all__ = [
-    "add_space_view",
     "AnnotationContext",
-    "arch",
     "Arrows3D",
-    "cmp",
+    "Clear",
     "DepthImage",
     "DisconnectedSpace",
-    "dt",
     "Image",
     "LineStrips2D",
     "LineStrips3D",
-    "log",
-    "new_blueprint",
     "Points2D",
     "Points3D",
     "SegmentationImage",
-    "set_auto_space_views",
-    "set_panels",
     "Tensor",
     "TextDocument",
     "TextLog",
     "Transform3D",
+    "add_space_view",
```

Comment:
killed alphabetical order by accident?

Reply:
this is literally the output of `sort` :grimacing: why I much prefer to alphabetical order tbh...